src/vue-plugins/http/module.js

```diff
@@ -93,21 +95,32 @@ export default {
           },
         });
       }
-
-      indices.forEach((index) => { // eslint-disable-line array-callback-return, max-len
+      return Promise.allSettled(
+        indices.map((index) => { // eslint-disable-line array-callback-return
         // Either send or post logs, depending on whether they originated on the server
         // Logs originating on the server possess an ID field; others do not.
-        const newLog = makeLog.toServer(rootState.farm.logs[index]);
-        farm().log.send(newLog, localStorage.getItem('token')) // eslint-disable-line no-use-before-define, max-len
-          .then(res => handleSyncResponse(res, index))
-          // .catch(err => handleSyncError(err, index, rootState, payload.router, commit));
-          .catch((err) => {
+          const newLog = makeLog.toServer(rootState.farm.logs[index]);
+          return farm().log.send(newLog, localStorage.getItem('token')) // eslint-disable-line no-use-before-define, max-len
+            .then(res => handleSyncResponse(res, index));
+        }),
+      )
+        .then((promises) => {
```

Comment:
This feels weird to me. Shouldn't we have a catch block? Maybe I need to become more familiar with how `allSettled` works, but something seems off.